dist/package.yaml

```diff
@@ -32,7 +32,7 @@ spec:
           default: ""
         stand_alone:
```

Comment:
Seems like this field is ripe for removal.

Reply:
You are right, I'd like to drop this.